man/readtags.1.rst.in

```diff
@@ -172,19 +173,34 @@ missing field is #f, so we could deal with missing fields:
                          (or (eq? $language "Python")\
                              (not $language)))' -l
 
-The ``member`` operator is used to deal with lists. Currently lists can't be
-built by hand, and they are only generated for the ``inherits:`` field. For
-example:
+``#/PATTERN/`` is for the case when string predicates (``prefix?``,
+``suffix``, and ``substr?``) are not enough. You can use "Posix
+extended regular expression" as PATTERN.
 
 * List all tags inherits from the class "A":
 
   .. code-block:: console
 
-     $ readtags -Q '(member "A" $inherits)' -l
+     $ readtags -Q '(#/(^|,)A(,|$)/ $inherits)' -l
+
+Here ``$inherits`` holds a comma-separated class list like "A,B,C",
```

Comment:
```suggestion
Here ``$inherits`` is a comma-separated class list like "A, B, C",
```

Reply:
I guess it never happens in the current implementation. However, I wonder we should fix the behavior of python parser... At least, as you wrote, the users must know the format inside inherits: are language-specific. I will write it to the man page or somewhere.